src/distribution/pert.rs

```diff
@@ -0,0 +1,339 @@
+use distribution;
+use source::Source;
+
+/// A PERT distribution.
+///
+/// The PERT distribution is a family of continuous probability distributions
+/// defined by the minimum (a), most likely (b) and maximum (c) values that
+/// a variable can take. It is a transformation of the four-parameter Beta
+/// distribution with an additional assumption that its expected value is
+/// `u = (a + 4 * b + c) / 6`.
+#[derive(Clone, Copy, Debug)]
+pub struct Pert {
```

Comment:
I think it should be called `PERT`, because the family is called so. What are your thoughts about this?

Reply:
done. was of two minds due to constants being upper case.